steps/cli/tkn.go

```diff
@@ -21,4 +24,30 @@ var _ = gauge.Step("Start and verify pipeline <pipelineName> with param <paramNa
 		pipelineRunName := tkn.StartPipeline(pipelineName, params, workspaces, "--use-param-defaults")
 		pipelines.ValidatePipelineRun(store.Clients(), pipelineRunName, "successful", "no", store.Namespace())
 	}
-})
\ No newline at end of file
+})
+
+var _ = gauge.Step("Start and verify dotnet pipeline <pipelineName> with values stored in variable <variableName> with workspace <workspaceValue>", func(pipelineName, variableName, workspaceValue string) {
+	values := store.GetScenarioDataSlice(variableName)
+	params := make(map[string]string)
+	workspaces := make(map[string]string)
+	workspaces[strings.Split(workspaceValue, ",")[0]] = strings.Split(workspaceValue, ",")[1]
+	paramName := "VERSION"
+	re := regexp.MustCompile(`\d+\.\d+`)
+	var exampleRevision string
+	for _, value := range values {
+		if value == "latest" {
+			continue
+		}
+		exampleRevision = re.FindString(value)
+		params[paramName] = value
+		versionInt, _ := strconv.ParseFloat(exampleRevision, 64)
```

Comment:
Will parsing as float work for any value?

Reply:
You mean, for any number?

Reply:
OK, looks safe